server/static/js/notebook.js

```diff
@@ -11,7 +11,14 @@ function initalizeNotebookMarkdown() {
     nb.markdown = function (text) {
         return mdRenderer.render(text)
     };
+
     nb.ansi = ansi_up.ansi_to_html
+
+    nb.display.html = nb.display.md_html
+    nb.display["text/html"] =  nb.display.html;
```

Comment:
I suppose it doesn't matter after the assignment on 17 - but it would be clearer to assign to `md_html`